examples/kms-alias/index.ts

```diff
@@ -23,7 +23,7 @@ const aliasAutoPrefixedWithAlias = new aws.kms.Alias("should-be-prefixed", {targ
 
 const aliasWithDedicatedName = new aws.kms.Alias("wont-be-autonamed", {
     targetKeyId: key.keyId,
-    name: "alias/my-decicated-name"
+    namePrefix: "alias/my-decicated-name-"
```

Comment:
It looks like this test might be intentionally testing setting a specific name. I can't think of a good reason for that though so this change looks good to me!

Reply:
I did think about generating a random name and using that for the `name` parameter, but I also couldn't think of a reason to keep it that way.

Alternatively we could test all three cases (auto-generated name, name, namePrefix)